server/neptune/workflows/internal/activities/execute.go

```diff
@@ -0,0 +1,12 @@
+package activities
+
+import "context"
+
+type executeCommandActivities struct{}
+
+type ExecuteCommandRequest struct {
+}
+
+func (t *terraformActivities) ExecuteCommand(ctx context.Context, request ExecuteCommandRequest) error {
```

Comment:
This is just for generic run commands yeah?

Reply:
Yes, I figured because it's configurable to be any kind of generic run command, there isn't a guarantee if it is deterministic, so I made it an activity.